src/_includes/templates/forms/form-eeo-issue.html

```diff
@@ -0,0 +1,160 @@
+                                     cf-icon
+                                     cf-icon-down"></span>
+                    </div>
+                </div>
+                <div class="form-l_col form-l_col-1-2">
+                    <label class="form-label-header" for="eeo_zip">
+                        Zip code*
+                    </label>
+                    <input class="input__long"
+                           id="eeo_zip"
+                           type="text"
+                           required>
+                </div>
+            </div>
+
+            <div class="form-l_col form-l_col-1-2">
+                <label class="form-label-header" for="eeo_telephone">
+                    Telephone Number*
+                </label>
+                <input class="input__long"
+                       id="eeo_telephone"
+                       type="text"
+                       required>
+            </div>
+
+            <div class="form-l_col form-l_col-1-2">
+                <label class="form-label-header" for="eeo_fax">
+                    Fax Number
+                </label>
+                <input class="input__long"
+                       id="eeo_fax"
+                       type="text"
+                       required>
+            </div>
+
+            <div class="form-l_col form-l_col-1-2">
+                <label class="form-label-header" for="eeo_email">
+                    Email Address*
+                </label>
+                <input class="input__long"
+                       id="eeo_email"
+                       type="text"
+                       required>
+            </div>
+
+            <div class="form-l_col form-l_col-1-2">
+                <label class="form-label-header" for="contact-time">
+                    Best time to reach you
+                </label>
+                <div class="custom-select">
+                    <select class="custom-select_select"
+                            data-placeholder="Select Time"
+                            id="contact-time">
+                          <option value="">Select time</option>
+                          {# TODO: Add additional times. #}
+                          <option>8:00 a.m</option>
+                     </select>
+                     <span class="custom-select_icon
+                                  cf-icon
+                                  cf-icon-down"></span>
+                </div>
+            </div>
+
+            <div class="form-l_col form-l_col-1">
+                <label for="eeo_complaint-desc">
+                    <span class="form-label-header">
+                        Nature of discrimination
+                    </span>
+                </label>
+                <textarea class="input__long"
+                          id="eeo_complaint-desc"
+                          required>
+                </textarea>
+            </div>
+
+            <div class="form-actions form-actions__right-on-med ">
```

Comment:
Add `form-l_col form-l_col-1` to classes to indent buttons so they are right-indent aligned with the form fields.